indexd/drs/blueprint.py

```diff
@@ -154,6 +155,9 @@ def indexd_to_drs(record, expand=False):
         "description": description,
```

Comment:
what about this?

Reply:
Previous description work from years back 
https://github.com/uc-cdis/indexd/commit/4902aea6aeb69f37b299dcd5e7df025f373f9882#diff-926a6ca016e96ef2492a0f1d27a5c220339f49274d0caea713c0403706869d34R120

This is just a commit by commit revert of all the work I've done.